### DISCUSSION TITLE
WorkFlow Github Build Error - With the link to the build

### DISCUSSION THREAD

**Discussion by GabrielCoderz** [perspective-dev/perspective#2492] [Q&A]
Sorry about the log file.

My intention was to try to show the error so that someone can help with the problem.

The idea is also to try to help the project with translation into Portuguese.

As I mentioned, there is an error that I cannot get past in the build phase. I did it on my own account, with the public repository.

The build with error is here: 
https://github.com/GabrielCoderz/perspective/actions/runs/7465271700

@texodus

**Comment by texodus** [comment]
I have no interest in translating the project to Portuguese.
